1.0/docs/start/first-element/step-3.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 3: Use data binding and properties"
+subtitle: Build your first Polymer element
+---
+
+
+Right now, the element doesn't do much. In this step, you'll give it a basic
+API, allowing you to configure the icon from markup, using an attribute, or
+from JavaScript, using a property.
+
+First, a bit of data binding. Find the `<iron-icon>` element and change the value of the `icon` attribute from `"polymer"` to  "`[[toggleIcon]]`".
+
+#### icon-toggle.html
+{: .caption }
+
+
+<pre class="prettyprint">
+&lt;!-- local DOM goes here -->
+&lt;iron-icon icon="[[toggleIcon]]">
+&lt;/iron-icon>
+</pre>
+
+Key information:
+
+  * `toggleIcon` is a <em>property</em> you'll define on the toggle button element. It doesn't have a default value
+yet.
+  * The `icon="[[toggleIcon]]" `assignment is a <em>data binding</em>. It links your element's `toggleIcon` <em>property</em> with the `<iron-icon>`'s `icon` property.
+
+You could now use your element and set the `toggleIcon` property in markup or using JavaScript, as shown in the following examples. <em>(You don't need to add this code to your project.)</em>
```

Comment:
When you add the toggleIcon binding in Plunker the demo immediately to favorite icons but it isn't clear why. I think this would be a good opportunity to mention the icon-toggle-demo.html file.

Alternatively... I'm not sure how much value that file is adding. Taking a look at it right now it feels like it's a layer of additional complexity. Would it be possible to just import the element into the index file and have the user at this stage actually write the line `<icon-toggle toggle-icon="star"></icon-toggle>`. I realize this is a big change but I'm feeling like the tutorial is doing too much for me